vscode-lean4/src/abbreviation/AbbreviationProvider.ts

```diff
@@ -11,6 +11,9 @@ export class AbbreviationProvider implements Disposable {
 	private readonly disposables = new Array<Disposable>();
 	private cache: Record<string, string | undefined> = {};
 
+    private abbreviationCompletedEmitter = new EventEmitter<TextEditor>();
```

Comment:
This event is unused now.

Reply:
True, but this is twice now that I've found it might be useful to have so I might just leave it for a bit and see what else pops up that needs this event.